api/users/userModel.js

```diff
@@ -0,0 +1,95 @@
+const db = require('../../data/db-config');
+
+const getAllUsers = async () =>
+  await db('users as u').select(
+    'u.id as user_id',
+    'u.first_name',
+    'u.last_name',
+    'u.school',
+    'u.bg_username',
+    'u.email',
+    'u.phone',
+    'u.created_at'
+  );
+
+const findUserByFilter = async (filter) =>
+  await db('users as u')
+    .where(filter)
+    .select(
+      'u.id as user_id',
+      'u.first_name',
+      'u.last_name',
+      'u.school',
+      'u.bg_username',
+      'u.email',
+      'u.phone',
+      'u.created_at'
+    );
+
+const findUserByID = async (id) =>
+  await db('users as u')
+    .where({ id })
+    .select(
+      'u.id as user_id',
```

Comment:
This looks like a bug: you are using "id" in the were but renaming to "user_id" in the select. I would keep it consistent.